rebar.config

```diff
@@ -29,7 +29,7 @@
 {profiles, [
             {test, [
                {deps, [
-                       {relcast, {git, "https://github.com/helium/relcast.git", {branch, "master"}}}
+                       {relcast, {git, "https://github.com/helium/relcast.git", {branch, "adt/otp-23"}}}
```

Comment:
Should we block this PR on https://github.com/helium/relcast/pull/41 and update this to a hash when that merges?

Reply:
it's a test only dep, but yes

Reply:
fixed this post relcast upgrade.